x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/pipeline_processors_editor.container.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FunctionComponent, useMemo } from 'react';
+
+import { Processor } from '../../../../common/types';
+
+import { deserialize } from './deserialize';
+
+import { useProcessorsState } from './processors_reducer';
+
+import { PipelineProcessorsContextProvider } from './context';
+
+import { OnUpdateHandlerArg } from './types';
+
+import { PipelineProcessorsEditor as PipelineProcessorsEditorUI } from './pipeline_processors_editor';
+
+export interface Props {
+  value: {
+    processors: Processor[];
+    onFailure?: Processor[];
+  };
+  onUpdate: (arg: OnUpdateHandlerArg) => void;
+  isTestButtonDisabled: boolean;
+  onTestPipelineClick: () => void;
+  learnMoreAboutProcessorsUrl: string;
+  learnMoreAboutOnFailureProcessorsUrl: string;
+  /**
+   * Give users a why to react to this component opening a flyout
```

Comment:
Typo?

```suggestion
   * Give users a way to react to this component opening a flyout
```

Reply:
Definitely a typo 😄. I'll leave the "why" up to the consumer code.